### ISSUE TITLE
Why does sample app import from angularjs?

### ISSUE THREAD

**Issue by mhand75** [ui-router/sample-app-angular-hybrid#26]
My understanding is that you should not be installing **uirouter/angularjs** for a hybrid app. All you need is the **uirouter/angular-hybrid**?

from sample app:
```
import uiRouter from uirouter/angularjs;
```

I followed the sample app when converting my app to a hybrid and it worked great. I am also using webpack. Later when I was doing some cleanup, I deleted what I thought were no longer needed folders (like the `@uirouter/angularjs`). When I saw this build error I changed it to pull from `@uirouter/core`. Is that correct? 

When I made this change, my app would no longer work. It looks like my vendor bundle is dramatically different after this change and I think something that was being included before is no longer being included (sorry for lack of details). When I switched back to @uirouter/angularjs everything works again, but I'm assuming this will probably cause me problems later?

Is the sample app right? If so, isn't it conflicting with the documentation?

I've used ui-router forever in angular 1.x apps and LOVE it. I'm hoping to continue using it for the hybrid apps we have. Thanks!

**Comment by christopherthielen** [comment]
> My understanding is that you should not be installing uirouter/angularjs for a hybrid app. All you need is the uirouter/angular-hybrid?

Correct, you only add `@uirouter/angular-hybrid` to your package.json.  However, `@uirouter/angular-hybrid` has dependencies on both `@uirouter/angular` and `@uirouter/angularjs`.  https://github.com/ui-router/angular-hybrid/blob/9924e13c03e630dc2d00b22adf2e424397aa46f3/package.json#L18-L24
... so you can also import directly from those packages.

```
import uiRouter from uirouter/angularjs;
```
This line in the sample app imports the angularjs module name (which is `ui.router`).

---

So to summarize, you only need `@uirouter/angular-hybrid` in your package.json but you can import code from `@uirouter/core`, `@uirouter/angular`, and `@uirouter/angularjs`

**Comment by christopherthielen** [comment]
Let me know if you have further questions!
